Bio/SeqIO/SnapGeneIO.py

```diff
@@ -209,7 +202,6 @@ def _parse_primers_packet(length, data, record):
     0x00: _parse_dna_packet,
     0x05: _parse_primers_packet,
     0x06: _parse_notes_packet,
-    0x09: _parse_cookie_packet,
```

Comment:
I would be inclined to leave ``_parse_cookie_packet`` in the dictionary as a form of documentation?

Reply:
The cookie should appear at the beginning of the file only. If we leave it in this dictionary, then the parser won't complain if a second cookie appears somewhere further in the file.

Reply:
If it is meant to be there once only, that's a good reason for this change.